lib/api/controllers/collectionController.js

```diff
@@ -261,7 +273,7 @@ class CollectionController {
       .then(response => {
         this.kuzzle.indexCache.add(request.input.resource.index, request.input.resource.collection);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 100 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)